spec.html

```diff
@@ -26993,13 +26993,12 @@ <h1>BigInt.prototype.toLocaleString ( [ _reserved1_ [ , _reserved2_ ] ] )</h1>
       <emu-clause id="sec-bigint.prototype.tostring">
         <h1>BigInt.prototype.toString ( [ _radix_ ] )</h1>
         <emu-note>
-          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ not present or is *undefined* the Number 10 is used as the value of _radix_.</p>
+          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ is *undefined* the Number 10 is used as the value of _radix_.</p>
```

Comment:
i'd prefer to separate out this kind of change from the intrinsic notation change, if possible.